main/handlersettingscommon.go

```diff
@@ -35,23 +30,20 @@ func settingsPath(configFolder string) (string, error) {
 	if err != nil {
 		return "", fmt.Errorf("Cannot find seqnum: %v", err)
 	}
-	return filepath.Join(configFolder, fmt.Sprintf("%d%s", seq, settingsFileSuffix)), nil
+	return filepath.Join(configFolder, fmt.Sprintf("%d%s", seq, ".settings")), nil
```

Comment:
Why was the constant dropped with inline string?

Reply:
reverted